### PR TITLE
IR-262: Keep manifest list children

### PR DESCRIPTION
**Pull request by dmage**

### CODE REVIEW

**Comment by flavianmissi** [comment]
nice!
/lgtm

File: pkg/cli/version/version.go [hunk 1]
```diff
@@ -86,8 +86,12 @@ func NewCmdVersion(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *co
 
 // Complete is copied from upstream version command with added clusteroperator client
 // to report OpenShift server version
-func (o *VersionOptions) Complete(f cmdutil.Factory, cmd *cobra.Command) error {
-	var err error
+func (o *VersionOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []string) error {
+	err := o.Options.Complete(f, cmd, args)
```
Comment (dmage) [inline pkg/cli/version/version.go:90]: This is a post-rebase fix. The upstream `Validate(...)` doesn't accept `args` anymore. Another solution is to copy the upstream implementation of `Validate(...)` as we did with `Complete(...)`.

**Comment by flavianmissi** [comment]
@dmage seems like unit tests are failing, is it legit?

**Review by soltysh — APPROVED** [review]
/approve

**Pull request merged**